config/default/migrate_plus.migration.hs_courses.yml

```diff
@@ -18,7 +18,7 @@ source:
   track_changes: true
   plugin: url
   data_fetcher_plugin: http
-  urls: /api/hs_courses
```

Comment:
So we are switching away from this approach? I forgot why we need to pull from the local endpoint in the first place.

Reply:
relative urls never worked. this is mostly for new sites that haven't configured their courses source or local development. its just putting in a url with an empty xml feed to prevent log errors. The reason we are pulling from local is to modify the courses xml data so that we can create unique guids on each course section and when there are no sections